ui/appui-react/src/appui-react/statusbar/popup/StatusBarPopover.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module StatusBar
+ */
+
+import * as React from "react";
+import { Popover } from "@itwin/itwinui-react";
+import { ButtonExpandIndicator } from "./ExpandIndicator";
+
+/** Popover component used in [[StatusBar]] component.
+ * This component should wrap the element that triggers the popover.
+ * @note Add the {@link StatusBarPopover.ExpandIndicator} to popover trigger buttons.
```

Comment:
@GerardasB do you think it's ok to have a note like this? I want to encourage users to use the expand indicator but not sure what is the best way to do that.

Reply:
I switched to using ` `` ` because I could not get the docs to work properly.